.circleci/config.yml

```diff
@@ -406,28 +406,8 @@ commands:
             .circleci/collect-service-logs.sh kube-system
           when: always
 
-      - run:
```

Comment:
We do this again lower, so I removed this one, as there is no need to collect the logs twice